_episodes/dataflow.md

```diff
@@ -17,3 +17,192 @@ learners should be able to
 __explain how a workflow document describes the input and output of a workflow and the flow of data between tools__
 
 {% include links.md %}
+
+
+
+CWL Workflows are about chaining different steps.
```

Comment:
I would suggest default instead to the definition given in our CWL documentation/user guide:   "CWL is a way to describe command line tools and connect them together to create workflows"